luaui/Widgets/gui_selfd_icons.lua

```diff
@@ -132,74 +165,105 @@ end
 function widget:DrawWorld()
 	if Spring.IsGUIHidden() then return end
```

Comment:
Would be nice to localize `Spring.IsGUIHidden` and `gl.DepthTest` as these are in drawworld callin

Reply:
done!